### PR TITLE
Changes to ps1 scripts and added Readme files, Also made changes to ARM templates and bicep

### PR DESCRIPTION
**Pull request by NikitaSingh-ms**

### CODE REVIEW

File: consent-for-mandatory-extension-with-Invoke-AzRestMethod/consent-mandatory-extension.ps1 [hunk 1]
```diff
@@ -1,6 +1,6 @@
-$subscriptionId = ""
-$resourceGroup = ""
-$tenantId = ""
+$subscriptionId = "680d0dad-59aa-4464-adf3-b34b2b427e8c"
```
Comment (priyjain358) [inline consent-for-mandatory-extension-with-Invoke-AzRestMethod/consent-mandatory-extension.ps1:1]: Please scrub off these values

File: enable-azure-hybrid-benfit-intent/arm template and bicep/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# AzSHCI-Automation
+
+Manage Azure Hybrid Benefit settings for Azure Stack HCI clusters using PowerShell and Azure CLI.
+
+Pre-requisites:
+	1. Create Workspace 'microsoft.operationalinsights/workspaces' manually before running script.
```
Comment (priyjain358) [inline enable-azure-hybrid-benfit-intent/arm template and bicep/README.md:6]: from where did this come?

File: enable-azure-hybrid-benfit-intent/arm template and bicep/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	2. Install Az Powershell https://learn.microsoft.com/en-us/powershell/azure/install-azure-powershell?view=azps-9.6.0
+
+Fill up the values in enable-wss.parameters.json:
```
Comment (priyjain358) [inline enable-azure-hybrid-benfit-intent/arm template and bicep/README.md:9]: nit:wss-parameters

File: enable-azure-hybrid-benfit-intent/arm template and bicep/enable-AHB-intent-parameters.json [hunk 4]
```diff
@@ -5,10 +5,7 @@
         "clusterNames": {
             "value": [
                 {
-                    "name": "test-clus-1"
```
Comment (priyjain358) [inline enable-azure-hybrid-benfit-intent/arm template and bicep/enable-AHB-intent-parameters.json:8]: these are test values, no need to remove them

File: enable-azure-hybrid-benfit-intent/scripts/enable_hybridbenefit.ps1 [hunk 5]
```diff
@@ -48,4 +47,6 @@ $clusterName = ""
 az stack-hci cluster extend-software-assurance-benefit `
                                     --name $clusterName `
```
Comment (priyjain358) [inline enable-azure-hybrid-benfit-intent/scripts/enable_hybridbenefit.ps1:48]: replace --name with --cluster-name

File: enable-wss-diagnostics/arm template and bicep/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# AzSHCI-Automation
+
+Manage Diagnostic Level and Windows Server Subscription settings for Azure Stack HCI clusters using PowerShell and Azure CLI.
+
+Pre-requisites:
+	1. Create Workspace 'microsoft.operationalinsights/workspaces' manually before running script.
```
Comment (priyjain358) [inline enable-wss-diagnostics/arm template and bicep/README.md:6]: why is this needed

File: install-ama-arc-extension/arm-template and bicep/README.md [hunk 7]
```diff
@@ -11,21 +11,20 @@ Pre-requisites:
 
 Fill up the values in ama-parameters.json: 
 
-1. cluster-names: array of the clusters to install AMA on 
-2. location: location where to create the DCR rule
```
Comment (priyjain358) [inline install-ama-arc-extension/arm-template and bicep/README.md:15]: why were 2 and 3 removed?

File: install-ama-arc-extension/arm-template and bicep/ama-parameters.json [hunk 8]
```diff
@@ -5,24 +5,18 @@
...
+					"name": ""
 				}
 			]
 		},
 		"location": {
 			"value": "East US"
 		},
 		"workspaceId": {
-			"value": "xxxxxx-xxxx-xxxx-xxx-xxxxxxxxxx"
+			"value": ""
```
Comment (priyjain358) [inline install-ama-arc-extension/arm-template and bicep/ama-parameters.json:16]: example values, can leave as it is

File: install-ama-arc-extension/scripts/README.md [hunk 9]
```diff
@@ -10,10 +10,8 @@ Update Variables in Scripts:
 		"workspaceId": "",
 		"workspaceResourceId": "" 
 		destinations: ""  <use the same name as given as step1>
-	2. update 4 variables in install_ama_extn
+	2. update 4 variables in install_mma_extn
```
Comment (priyjain358) [inline install-ama-arc-extension/scripts/README.md:13]: should be ama?

File: install-mma-arc-extension/arm template and bicep/README.md [hunk 10]
```diff
@@ -1,31 +1,31 @@
...
-1. cluster-names: array of the clusters to install AMA on 
+1. cluster-names: array of the clusters to install MMA on 
 2. location: location where to create the DCR rule 
 3. Workspace-id, workspace-name: details of the workspace created above
 
 
 To run the ARM-template: 
 
-New-AzResourceGroupDeployment -Name <deployment-name> -ResourceGroupName <resource-group> -TemplateFile .\install-ama.json -TemplateParameterFile .\ama-parameters.json
+New-AzResourceGroupDeployment -Name <deployment-name> -ResourceGroupName <resource-group> -TemplateFile .\mma_template.json -TemplateParameterFile .\mma_parameters.json
```
Comment (priyjain358) [inline install-mma-arc-extension/arm template and bicep/README.md:21]: the file name is install_mma.json, no?

File: install-mma-arc-extension/arm template and bicep/mma_template.bicep [hunk 11]
```diff
@@ -1,8 +1,10 @@
-var clusterNames = ['cluster1', 'cluster2']
-
+//var clusterNames = ['cluster1']
+param clusterNames array
```
Comment (priyjain358) [inline install-mma-arc-extension/arm template and bicep/mma_template.bicep:2]: hope you have verified these changes?

File: install-mma-arc-extension/script/install_mma.ps1 [hunk 12]
```diff
@@ -1,36 +1,81 @@
...
 $clusters = @(Get-AzStackHciCluster -ResourceGroupName $resourceGroup).Name
 
+ 
+
+
 foreach($cluster in $clusters) {
-    Start-Job -ScriptBlock {
-        Write-Host ("Installing MMA extension for cluster $using:cluster")
-        New-AzStackHciExtension -ClusterName $using:cluster -ResourceGroupName $using:resourceGroup -ArcSettingName "default" -ExtensionName "MicrosoftMonitoringAgent" -AutoUpgrade $true -Publisher "Microsoft.EnterpriseCloud.Monitoring" -Type "MicrosoftMonitoringAgent" -SettingsFile $using:settings -ProtectedSettingsFile $using:protectedSetting -TypeHandlerVersion "1.10"
+     Start-Job -ScriptBlock{
```
Comment (priyjain358) [inline install-mma-arc-extension/script/install_mma.ps1:71]: remove the script block

File: install-wac-in-portal-arc-extension/scripts/install-wac.ps1 [hunk 13]
```diff
@@ -2,31 +2,68 @@ $subscriptionId = ""
...
-$connectivityPath = "path to connectivity.json"
+ 
+
+$settingsPath=@{
+    "port"= "6516"
+}
+
+ 
+
+$connectivityPath=@{
```
Comment (priyjain358) [inline install-wac-in-portal-arc-extension/scripts/install-wac.ps1:13]: rename the variable since its not a path anymore

File: install-wac-in-portal-arc-extension/scripts/install-wac.ps1 [hunk 14]
```diff
@@ -2,31 +2,68 @@ $subscriptionId = ""
 $resourceGroup = ""
 $tenantId = ""
 
-$settingsPath = "path to port-wac.json"
-$connectivityPath = "path to connectivity.json"
+ 
+
+$settingsPath=@{
```
Comment (priyjain358) [inline install-wac-in-portal-arc-extension/scripts/install-wac.ps1:7]: rename the variable since its not a path anymore

File: enable-azure-hybrid-benfit-intent/arm template and bicep/README.md (hunk 2 above)
Comment (NikitaSingh-ms) [reply enable-azure-hybrid-benfit-intent/arm template and bicep/README.md:6]: There was a similar Readme file for ama installation, so to have consistency I added these. If they are unnecessary I can remove them.

File: enable-wss-diagnostics/arm template and bicep/README.md (hunk 6 above)
Comment (NikitaSingh-ms) [reply enable-wss-diagnostics/arm template and bicep/README.md:6]: There was a similar Readme file for ama installation, so to have consistency I added these. If they are unnecessary, I can remove them.

File: install-mma-arc-extension/arm template and bicep/README.md (hunk 10 above)
Comment (NikitaSingh-ms) [reply install-mma-arc-extension/arm template and bicep/README.md:21]: In the folder structure template file name is mma_template.json.

File: install-mma-arc-extension/arm template and bicep/mma_template.bicep (hunk 11 above)
Comment (NikitaSingh-ms) [reply install-mma-arc-extension/arm template and bicep/mma_template.bicep:2]: Yes

File: install-mma-arc-extension/script/install_mma.ps1 (hunk 12 above)
Comment (NikitaSingh-ms) [reply install-mma-arc-extension/script/install_mma.ps1:71]: Ok, should I remove using: as well?

Comment (priyjain358) [reply install-mma-arc-extension/script/install_mma.ps1:71]: yes

**Review by priyjain358 — APPROVED** [review]
